shallow_backup/printing.py

```diff
@@ -37,6 +37,22 @@ def print_green_bold(text):
 	print(Fore.GREEN + Style.BRIGHT + text + Style.RESET_ALL)
 
 
+def print_blue_bold_path(text, path):
```

Comment:
Let's just cut the word `bold` out of the function names since not everything we're printing is bold, and then let's move the color to the end so it's uniform.

i.e. `print_path_COLOR()` as opposed to `print_COLOR_path()`

```suggestion
def print_path_blue(text, path):
```